docs/compatibility.md

```diff
@@ -46,8 +46,7 @@ These are repos that can be started as a some kind of process, such as a web app
       * **TODO: Missing support for running the worker**
    - ✅ govuk_publishing_components
    - ✅ govuk-developer-docs
-   - ⚠ hmrc-manuals-api
-      * **TODO: Missing support for a webserver stack**
+   - ✅ hmrc-manuals-api
```

Comment:
I'm not too sure what you mean by the rationale for this not needing a webserver stack? I'd have thought this one was as relevant as other apps that are flagged as not having a webserver stack as the interface to this app isn't exposed so a developer can only run tests?

Reply:
That was the original reasoning. However, with this app I paused to think: would someone ever need the app stack? This app is quite special: it's an API app that's only called externally. On the other hand, the other apps are either internal dependencies or have a GUI that someone may want to develop against. Bouncer is similar too, actually.

I don't really mind either way. This file is somewhat vague in that the compatibility will always be superficial and certain features may not work. Here I'm pushing to only flag missing pieces that would clearly add value. That said, it wouldn't be hard to add and test an app stack for these apps, so I don't feel strongly either way.

What do you think?

Reply:
> would someone ever need the app stack? 

I'd have been expecting that when you're working with this API you want a version running that you can send requests to - through postman, curl or similar. I recall when I worked a lot with the Publishing API I'd be spending a lot of my time with postman open. Otherwise any experimentation with it has to be done through the test suite, which seems unusual.

My logic for thinking it should have `**TODO: Missing support for a webserver stack**` is that this text implies that the interface to a running server of the app isn't available - it doesn't seem specific to whether that interface is graphical or not. So precedent of the tick would indicate a server of the app is available.

Reply:
> when you're working with this API you want a version running that you can send requests to

This is what I'm sceptical of. But I haven't worked with it myself, so it's just a hunch.

>So precedent of the tick would indicate a server of the app is available.

That's true: it makes less sense looking at it in reverse like that. I've removed the commit, as it's not a big deal either way: someone can just go and do the work to add the missing piece when it's eventually needed.

Reply:
Cool thanks - I guess the real problem is we have an app that virtually no-one works with 😞